src/main/cpp/CMakeLists.txt

```
@@ -196,6 +196,7 @@ add_library(
   src/HashJni.cpp
   src/HistogramJni.cpp
   src/HostTableJni.cpp
+  src/HLLPPJni.cpp
```

Let's try to be generic.
```suggestion
  src/AggregationJni.cpp
```

---

Renamed to: HLLPPHostUDFJni
AggregationJni is too generic